src/main/java/org/wildfly/security/auth/provider/FileSystemSecurityRealm.java

```diff
@@ -431,6 +435,15 @@ public void setAttributes(final Attributes attributes) throws RealmUnavailableEx
             replaceIdentity(newIdentity);
         }
 
+        @Override
+        public Attributes getAttributes() throws RealmUnavailableException {
+            final LoadedIdentity loadedIdentity = loadIdentity(false, false);
```

Comment:
Actually that was a lie, this one does not call replaceIdentity() :-) I can fix that